sherpa/astro/fake.py

```diff
@@ -21,111 +21,158 @@
 The ``fake_pha`` routine is used to create simulated
 `sherpa.astro.data.DataPHA` data objects.
 '''
+
+from functools import reduce
+import operator
```

Comment:
Where are those imports used?

Reply:
These are left over from something (maybe the RNG changes or maybe some attempt I made to be a bit-more functional in the code). I'll remove them (and the other files).